### ISSUE TITLE
close tooltips when zooming, panning, grabbing (1.0.1)

### ISSUE THREAD

**Issue by pmbert** [chartjs/chartjs-plugin-zoom#550]
I think it would be good to close the tooltip box when zooming, panning, grabbing....

**Comment by Peter-H-Etteplan** [comment]
I agree that would be a nice improvement. For now I employed this solution which works pretty well:
`      onPanStart: ({ chart }) => {
        chart.config.options.plugins.tooltip.enabled = false
      },
      onPanComplete: ({ chart }) => {
        chart.config.options.plugins.tooltip.enabled = true
        chart.update()
      },
`
Without `update()` the tooltip would not come back until e.g. the next time I zoomed. An ugly thing is that when the pan completes and the tooltip is shown again, it's not shown at the current mouse position but at the same point/position as when the panning started, so you need to nudge the mouse a bit for it to jump into place.

**Comment by estanglerbm** [comment]
Calling chart.update() [to force the tooltip.enable = true to take effect] in onPanComplete() / onZoomComplete() can also cause temporary drawing issues on the edges of the chart during transition, like zooming (due to performance?)

Better to set a flag and then call chart.update() during onHover() and reset flag.
